imls-backend/test/.test-config

```diff
@@ -0,0 +1,4 @@
+export PYTEST_SCHEME="http"
+export PYTEST_HOSTNAME="localhost"
+export PYTEST_PORT=3000
+export PYTEST_JWT_SECRET="${PGRST_JWT_SECRET}"
```

Comment:
I don't have a huge opinion here, but just thought i'd ask -- could these be in a .env file or do they need to be in this kind of file instead?

Reply:
They're both unit test-specific and fine to check into the repo, unlike the (SHHHH) secrets contained in .env. If they were put in .env you'd have to a) have people manually add them since .env isn't in the repo, and b) would result in 4 environmental variables that have no use outside of pytest.